public/auth/signin.html

```diff
@@ -1,8 +1,12 @@
-<div class="signin" ng-controller='authController'>
+<div class="signin row" ng-controller='authController'>
 
   <div class='login facebook col-lg-12'>
-  <image class="pokeball" src="assets/pokeball.png" x="0" y="0"  width="400px" height="400px" />
-	 <a class = "col-lg-12" href="/signin/facebook" target="_self"><img src="http://i.stack.imgur.com/Vk9SO.png"></a> 
+  	<image class="pokeball" src="assets/pokeball.png" x="0" y="0"  width="400px" height="400px" />
+	 	<a class = "col-lg-12" href="/signin/facebook" target="_self"><img class='fbbutton' src="http://i.stack.imgur.com/Vk9SO.png"></a> 
+		<div class="credits">
+			<p>This website is a board game application based on the Pokemon Master Trainer Board Game. The authors of this site does not take any credit for the board game made by Milton Bradley nor do they take credit for any of the artwork on displayed on the page.</p>
```

Comment:
Replace with:
This application is a recreation of the Pokemon Master Trainer Board Game (which is no longer in production). The authors of this site takes no credit for the board game made by Milton Bradley nor do they take credit for any of the artwork displayed on the page.